client/src/components/Homepage/style.css

```diff
@@ -0,0 +1,46 @@
+.statusdiv {
+  display: flex;
+  height: 51px;
+  position: absolute;
+}
+.pendingDiv {
+  display: flex;
+  margin-right: 5%;
+  text-align: center;
+  font-family: Roboto;
+  font-style: normal;
+  font-size: medium;
+  letter-spacing: 0.06vw;
+  color: #363936;
+  padding: 14px 0 0 0;
+}
+.closedDiv {
+  display: flex;
+  text-align: center;
+  font-family: Roboto;
+  font-style: normal;
+  font-size: medium;
+  letter-spacing: 0.06vw;
+  color: #363936;
+  padding: 14px 0 0 0;
+}
+.selected {
+  background: var(--main-color, blue);
+  color: #ffffff;
+}
+.status {
+  margin: 0 18px;
+  cursor: pointer;
+}
+.circle {
+  margin-top: 0px;
```

Comment:
collapse these margin tags into ```margin: 0 10 0 -11```

Reply:
this is a performance optimization so i'm not sure if we need it, but it's worth to point it out.